Look like you cannot add something in "[]" to title:
```
[ERROR] Error: Can't process doc metadata for doc at path path=/opt/clickhouse-docs/docs/en/engines/table-engines/integrations/materialized-postgresql.md in version name=current
    at processDocMetadata (/opt/clickhouse-docs/node_modules/@docusaurus/plugin-content-docs/lib/docs.js:200:15)
    at async Promise.all (index 1137)
```

--- a/docs/en/engines/table-engines/integrations/materialized-postgresql.md
+++ b/docs/en/engines/table-engines/integrations/materialized-postgresql.md
@@ -2,8 +2,9 @@
 slug: /en/engines/table-engines/integrations/materialized-postgresql
 sidebar_position: 130
 sidebar_label: MaterializedPostgreSQL
-title: [Experimental] MaterializedPostgreSQL
 ---
+
+# [experimental] MaterializedPostgreSQL
 
 Creates ClickHouse table with an initial data dump of PostgreSQL table and starts replication process, i.e. executes background job to apply new changes as they happen on PostgreSQL table in the remote PostgreSQL database.
 

